codespell_lib/data/dictionary.txt

```diff
@@ -16802,6 +16882,7 @@ javascropt->javascript
 javasript->javascript
 javasrript->javascript
 jave->java, have,
+JaveScript->JavaScript
```

Comment:
Not necessary to have this capitalized, right?

Reply:
Which side do you mean, right or left? `JavaScript` is the official correct form.

Reply:
Just wondering why the left one is capitalized?

Reply:
Not really sure...
```suggestion
javescript->JavaScript
```

Reply:
The left side is, and will likely always be, lower cased (or case ignored) by the code. We've got a lot where it matches the RHS and I don't think that's a bad thing. It's probably a lower cognitive load to compare them (and spot the change/typo) if it's the same case. But I don't think it makes much difference.